app/client/questions/add_question.coffee

```diff
@@ -108,18 +111,23 @@ Template.add_question.events
 
     question =
       text: formData.text
-      question_type: instance.type.get()
+      questionType: instance.type.get()
       properties: questionProperties
 
-    console.log question
-    Meteor.call "addQuestion", instance.form._id, question, (error, response) ->
-      if error
-        toastr.error("Something went wrong")
-      else
-        form.reset()
-        instance.choices.find().forEach ({_id})->
-          instance.choices.remove(_id)
-        instance.type.set('inputText')
-        toastr.success('Question added')
+    Meteor.call "createQuestion", instance.form.id, question,
+      (error, questionId) ->
+        if error
+          toastr.error error.message
+        else
+          form.reset()
+          instance.choices.find().forEach ({_id})->
+            instance.choices.remove _id
+          instance.type.set 'inputText'
+          lastItemOrder = instance.questions.findOne({}, sort: {order: -1}).order
```

Comment:
I believe the exception is thrown here due to questions is undefined on initial page load